src/main/java/net/william278/velocitab/config/Settings.java

```diff
@@ -16,37 +22,84 @@
         ┗╸ Placeholders: %players_online%, %max_players_online%, %local_players_online%, %current_date%, %current_time%, %username%, %server%, %ping%, %prefix%, %suffix%, %role%""")
 public class Settings {
 
-    @YamlKey("header")
-    private String header = "&rainbow&Running Velocitab v" + BuildConstants.VERSION + " by William278";
-    @YamlKey("footer")
-    private String footer = "[There are currently %players_online%/%max_players_online% players online](gray)";
-    @YamlKey("format")
-    private String format = "&7[%server%] &f%prefix%%username%";
-    @YamlKey("excluded_servers")
-    private ArrayList<String> excludedServers = new ArrayList<>();
-    @YamlKey(("update_rate"))
+    @YamlKey("headers")
+    @YamlComment("The header to display on the tab list per group.")
+    private Map<String, String> headers = Map.of("default", "&rainbow&Running Velocitab v" + BuildConstants.VERSION + " by William278");
+    @YamlKey("footers")
+    @YamlComment("The footer to display on the tab list per group.")
+    private Map<String, String> footers = Map.of("default", "[There are currently %players_online%/%max_players_online% players online](gray)");
+    @YamlKey("formats")
+    @YamlComment("How the player will appear on the tab list per group.")
+    private Map<String, String> formats = Map.of("default", "&7[%server%] &f%prefix%%username%");
+    @YamlKey("server_groups")
+    @YamlComment("The servers in each group\nAll servers not defined in a group will be excluded from Velocitab")
```

Comment:
```suggestion
    @YamlComment("The servers in each group of servers")
```
i'd rather keep this in one line